cvmfs/network/download.cc

```diff
@@ -70,6 +70,29 @@ using namespace std;  // NOLINT
 
 namespace download {
 
+bool Interrupted(JobInfo *info, const std::string &fqrn) {
```

Comment:
Style: output parameter last
```suggestion
bool Interrupted(const std::string &fqrn, JobInfo *info) {
```